drivers/nrf24l01p/nrf24l01p.c

```diff
@@ -189,12 +199,10 @@ int nrf24l01p_on(nrf24l01p_t *dev)
     char read;
     int status;
 
-    gpio_clear(dev->cs);
     nrf24l01p_read_reg(dev, REG_CONFIG, &read);
     hwtimer_spin(DELAY_CS_TOGGLE_TICKS);
```

Comment:
@gebart, I assume they are not. They have been introduced in #1704 in cause of "inexplicable" timing errors. Just today we found one possible reason for this in #2315. I will test this soon but I have not time for this today.